packages/react-router/lib/hooks.tsx

```diff
@@ -115,7 +115,7 @@ export function useNavigationType(): NavigationType {
 }
 
 /**
- * Returns true if the URL for the given "to" value matches the current URL.
+ * Returns a PathMatch object if the given pattern matches the current URL.
```

Comment:
Should include a description of what happens when no match is found. I.e. ", else `null`."